cypress/selectorHelpers/index.jsx

```diff
@@ -0,0 +1,24 @@
+const selectDataHook = hook => `[data-test="${hook}"]`; // writes to component
+const writeDataHook = hook => ({ "data-test": hook }); // writes to selectors file
+
+// Adds objects with data-qahooks properties to be passed into React component props
+// i.e. { key: { data-qahook: <some_qa_hook> }, ... }
+
+const writeSelectorGenerator = Selectors =>
```

Comment:
Let's remove this old selectorHelpers folder now since the other branch replaces it